### PR TITLE
readonly inputs

### LINKED ISSUE #387: Style readonly inputs
**Please describe what has to be done**
For example, on the account page, the username field is read only as we do not support changing usernames, but still want the user to be able to see what it is. We're setting the `readOnly` attribute on the field but it still looks like you could interact with it, even though you cant. 

**Additional context**
Perhaps the purple outline that it creates turns red instead? I think you can even turn the cursor to a crossed out circle or something.

### PR DESCRIPTION
**Pull request by Isaac-the-Man**
### Description of change(s), including why:
Styled readonly inputs to make it clear that it's un-editable.

### Type of change(s)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Any work to still do or other concerns:

- (none)

### Closing issues:

- Closes #387 

### Screenshots (if frontend change) of before and after:
Original
original
Original Focused
original_focused
After
after
After Focused
after_focused

### CODE REVIEW

**Review by neha-deshpande001 — APPROVED** [review]

**Review by hbh7 — APPROVED** [review]
Love it!

**Comment by hbh7** [comment]
Even though this has all it's reviews, I want to wait until the next meeting to merge it so that everyone gets a chance to see it and an opportunity for comments.

**Pull request merged**
